acquia_lift.module

```diff
@@ -813,191 +707,24 @@ function acquia_lift_personalize_option_set_save($option_set) {
         }
         if (visitor_actions_save_action($action)) {
           personalize_goal_save($option_set->agent, $action_name, 1);
-          $needs_starting = TRUE;
         }
       }
     }
   }
-  if ($needs_starting && $field['settings']['personalize']['auto_start']) {
-    // We also need to make sure the agent is set to running after everything
-    // has been sync'd up.
-    $queue = DrupalQueue::get('acquia_lift_sync');
-    $queue->createItem(array(
-      'callback' => 'personalize_agent_set_status',
-      'args' => array(
-        $agent->machine_name,
-        PERSONALIZE_STATUS_RUNNING
-      )
-    ));
-    // Mark this agent as "pending" as it will be started once the queue runs.
-    acquia_lift_set_pending_status($agent->machine_name, $agent->label);
-  }
 }
 
 /**
  * Implements hook_personalize_option_set_delete().
  */
 function acquia_lift_personalize_option_set_delete($option_set) {
```

Comment:
I was thinking we might need to do something but then never got around to it, will remove it for now.